src/openrct2/ride/Ride.cpp

```diff
@@ -1473,45 +1472,45 @@ static int32_t RideGetNewBreakdownProblem(const Ride& ride)
     // Brakes failure can not happen if block brakes are used (so long as there is more than one vehicle)
     // However if this is the case, brake failure should be taken out the equation, otherwise block brake
     // rides have a lower probability to break down due to a random implementation reason.
-    if (ride.IsBlockSectioned())
-        if (ride.NumTrains != 1)
+    if (ride.isBlockSectioned())
+        if (ride.numTrains != 1)
             return -1;
 
     // If brakes failure is disabled, also take it out of the equation (see above comment why)
     if (GetGameState().Cheats.disableBrakesFailure)
         return -1;
 
-    auto monthsOld = ride.GetAge();
-    if (monthsOld < 16 || ride.reliability_percentage > 50)
+    auto monthsOld = ride.getAge();
+    if (monthsOld < 16 || ride.reliabilityPercentage > 50)
         return -1;
 
     return BREAKDOWN_BRAKES_FAILURE;
 }
 
-bool Ride::CanBreakDown() const
+bool Ride::canBreakDown() const
 {
-    if (GetRideTypeDescriptor().AvailableBreakdowns == 0)
+    if (getRideTypeDescriptor().AvailableBreakdowns == 0)
     {
         return false;
     }
 
-    const auto* entry = GetRideEntry();
+    const auto* entry = getRideEntry();
     return entry != nullptr && !(entry->flags & RIDE_ENTRY_FLAG_CANNOT_BREAK_DOWN);
 }
 
 static void ChooseRandomTrainToBreakdownSafe(Ride& ride)
 {
     // Prevent integer division by zero in case of hacked ride.
-    if (ride.NumTrains == 0)
+    if (ride.numTrains == 0)
         return;
 
-    ride.broken_vehicle = ScenarioRand() % ride.NumTrains;
+    ride.brokenTrain = ScenarioRand() % ride.numTrains;
```

Comment:
This is a subtle terminology change I hadn't noticed until now. I suppose this was changed to be more consistent with `numTrains`?

Reply:
Yes, vehicle is very ambiguous as it can refer to either train or car. We already changed a few instances, I just continued that trajectory.